src/screens/Home/components/SendTokenNotification.tsx

```diff
@@ -0,0 +1,205 @@
+    (async () => {
+      if (!hash && batchHash) {
+        const submitedBatchHash = await etherspotService.waitForTransactionHashFromSubmittedBatch(chain, batchHash);
+        setHash(submitedBatchHash);
+        setTimerStatus(STATUS.STOPPED);
+        dispatch(fetchAllAccountsAssetsBalancesAction());
+      }
+    })();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  if (!isVisible || !data?.isSuccess) return null;
+
+  const viewOnBlockchain = () => {
+    if (!hash) return;
+    dispatch(
+      viewTransactionOnBlockchainAction(chain, {
+        hash,
+        batchHash: batchHash,
+        fromAddress: contractAddress,
+      }),
+    );
+  };
+
+  const onClose = () => {
+    setIsVisible(false);
+    setTimerStatus(STATUS.STOPPED);
+  };
+
+  const NormalTitle = () => {
+    if (type === TRANSACTION_TYPE.SEND) {
+      return (
+        <Title>
+          {t('transactionNotification.sending') + ' ' + amount + ' ' + data.symbol}
+          <NormalIcon nameOrUrl={assetData?.iconUrl} isUrl />
+          <SubText>{'(' + value + ') ' + t('transactionNotification.on')}</SubText>
+          <NormalIcon nameOrUrl={chain} />
+        </Title>
+      );
+    }
+    return (
+      <Title>
+        {t('transactionNotification.sending') + ' ' + value}
+        <SubText>{' ' + t('transactionNotification.on')}</SubText>
+        <NormalIcon nameOrUrl={chain} />
+      </Title>
+    );
+  };
+
+  return (
+    <TouchableContainer key={index} onPress={viewOnBlockchain}>
+      <HorizontalContainer>
+        <Summary>
+          <HorizontalSubContainer style={{ paddingBottom: 5 }}>
+            <Icon
+              name={hash ? 'checkmark-circle' : 'pending-process'}
+              width={24}
+              height={24}
+              style={{ paddingRight: 5 }}
+            />
+            {hash ? <Title>{t('transactionNotification.you_sent')}</Title> : <NormalTitle />}
+          </HorizontalSubContainer>
+          <HorizontalSubContainer>
+            <Title numberOfLines={1} style={{ marginLeft: 29 }}>
+              {t('transactionNotification.to') + ' '}
+              <SubText>
+                {to.slice(0, 10)}...{to.slice(32, 42)}
+              </SubText>
+            </Title>
+          </HorizontalSubContainer>
+        </Summary>
+        <ButtonContainer onPress={onClose}>
+          <Icon color={colors.control} name={'close'} width={30} height={30} style={{}} />
+        </ButtonContainer>
+      </HorizontalContainer>
+      <Line />
+      <HorizontalContainer>
+        <SubText style={{ width: '30%' }}>{t('transactionNotification.time') + ': ' + time}</SubText>
+        <SubText style={{ flex: 1 }}>{t('transactionNotification.gas') + ': ' + gasValue}</SubText>
+        <ButtonContainer onPress={viewOnBlockchain}>
+          <Icon color={colors.control} name={'info'} width={24} height={24} style={{}} />
+        </ButtonContainer>
+      </HorizontalContainer>
+    </TouchableContainer>
```

Comment:
This looks quite similar to the exchange notification. Could this be a shared view component?

Reply:
I made some shared view component